CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree Android Drop-In Release Notes
 
+## unreleased
+
+* Notify merchant via error message the payment method is duplicated (fixes #357)
```

Comment:
Should we add a line for `* Bump braintree_android module dependency versions to 4.15.0` since that's being done here as well?

Reply:
Yup good call 👍